ops/log.py

```diff
@@ -11,16 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
It's very much a nit where I earlier said maybe we should add in the blank line, but here I'm very -1 on actually removing one. But maybe we just need the team discussion and a style guide ruling and everyone explains to me why I'm wrong :)